cmd/minikube/cmd/start_flags.go

```diff
@@ -83,6 +83,15 @@ const (
 	imageRepository         = "image-repository"
 	imageMirrorCountry      = "image-mirror-country"
 	mountString             = "mount-string"
+	mount9PVersion          = "mount-9p-version"
+	mountGID                = "mount-gid"
+	mountIPFlag             = "mount-ip"
```

Comment:
Some have "Flag" at the end and some do not. Are all of these flags? Could we drop "Flag" for consistency between these?

Reply:
`mountIP`, `mountPort` & `mountType` are already defined in `mount.go` so I had to differentiate them

Reply:
Alright, works for me.